controller/src/main/java/org/jboss/as/controller/logging/ControllerLogger.java

```diff
@@ -3444,4 +3446,36 @@ XMLStreamException discoveryOptionsMustBeDeclared(String adminOnlyCmd, String po
 
     @Message(id = 422, value = "Could not load module '%s' for transformers")
     RuntimeException couldNotLoadModuleForTransformers(String name, @Cause ModuleLoadException e);
+
```

Comment:
Can you add a comment that a bunch of ids were deliberately skipped? That will make it less mysterious a few years from now.